specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/examples/HostPools_Update_Post.json

```diff
@@ -26,6 +25,26 @@
     }
   },
   "responses": {
+    "200": {
+      "body": {
+        "warnings": [
+          {
+            "faultType": "ServiceError",
```

Comment:
In general, fault is equivalent to error; this makes me a bit confused here. Normally I just see "systemCode" instead. are you sure we want to name it as faultCode?

Reply:
its not my design. But sheikh and shaila decided on it.

Reply:
@datr93  would you please ask Shaila/Sheikh to reply here.  thanks.